docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -554,14 +554,17 @@ There are three table properties available for use in table creation.
     - Set partition columns.
   * - ``checkpoint_interval``
     - Set the checkpoint interval in seconds.
+  * - ``change_data_feed_enabled``
+    - Enables storing change data feed entries for delta lake.
```

Comment:
Indicates whether change data feed entries are to be stored for the write operations

Reply:
I am not sure that sentence would be correct, remember that for INSERTs, that are writes, we don't store anything.